src/components/EReceiptThumbnail.tsx

```diff
@@ -83,6 +87,7 @@ function EReceiptThumbnail({transaction}: EReceiptThumbnailProps) {
                 styles.overflowHidden,
                 styles.alignItemsCenter,
                 containerHeight && containerHeight < variables.eReceiptThumnailCenterReceiptBreakpoint ? styles.justifyContentCenter : {},
+                borderRadius ? {borderRadius} : {},
```

Comment:
Why do we add the border-radius here?

Reply:
Before we used to give the image component border-radius now to achieve the same we need to give the border radius view inside `EReceiptThumbnail`.